File: Python/lt_01_estseq/lt01_estseq_21.py
nota = 0.0
for i in range(0,4,1):
    nota+=((float(input("Insira a "+(str(i+1)) +"º nota Bimestral: ")))/4)

if nota>=6.0:
    print("APROVADO - NOTA:",nota)
elif nota>=3.0:
    print("EXAME - NOTA:",nota)
else:
    print("REPROVADO - NOTA:",nota)

File: C/lt_01_estseq/lt01_estseq_21.c
#include <stdio.h>

int main() {

  float nota, aux;
  int i, j;

  for (i = 0; i < 1; i++) {

    nota = 0;
    for (j = 0; j < 4; j++) {

      printf("Insira o valor da %d nota do aluno %d: ", j + 1, i + 1);
      scanf("%f", &aux);
      nota += aux;
    }

    nota /= j;

    if (nota >= 6) {
      printf("Aluno %d - APROVADO.\n", i + 1);
    } else if (nota >= 3) {
      printf("Aluno %d - EXAME.\n", i + 1);
    } else {
      printf("Aluno %d - RETIDO.\n", i + 1);
    }
  }
  return 0;
}
